### DISCUSSION TITLE
minimal function in constraint

### DISCUSSION THREAD

**Discussion by HQBao1994** [yalmip/YALMIP#1154] [Optimization modelling] [answered]
Dear Pf. Johan
 hello, I have a MPC problem, but there exist a minimal function in constraint. The details is in the picture.
The constraint 4e means  function min K(s) < 0 for some s in (0,1).
 But I have no ideal of how to model it in yalmip. I hope you can give me some idea. Thanks a lot!
![image](https://user-images.githubusercontent.com/110264585/181854787-b17587b0-1c1f-46f4-b12c-e2235e532bf2.png)

**Comment by johanlofberg** [comment, marked as answer]
Responded on https://groups.google.com/g/yalmip/c/kJEGz6t_duc
